### PR TITLE
Add Service rocketchat

### PR DESCRIPTION
**Pull request by kha7iq**
Changelog:

    Implement RocketChat service
    Update readme to include the service.
    Add usage example

Closes: https://github.com/nikoksr/notify/issues/26

### CODE REVIEW

File: service/rocketchat/rocketchat.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	"github.com/RocketChat/Rocket.Chat.Go.SDK/models"
+	"github.com/RocketChat/Rocket.Chat.Go.SDK/rest"
+	"github.com/pkg/errors"
+)
+
+// RocketChat struct holds necessary data to communicate with the RocketChat API.
+type RocketChat struct {
+	client  *rest.Client
+	chNames []string
```
Comment (nikoksr) [inline service/rocketchat/rocketchat.go:15]:
```suggestion
	channelNames []string
```

File: service/rocketchat/rocketchat.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		client:  c,
+		chNames: []string{},
+	}
+
+	return &rc, nil
+}
+
+// AddReceivers takes rocketchat channel names and adds them to the internal channel list. The Send method will send
+// a given message to all channels in the list.
+func (r *RocketChat) AddReceivers(chNames ...string) {
```
Comment (nikoksr) [inline service/rocketchat/rocketchat.go:46]:
```suggestion
func (r *RocketChat) AddReceivers(channelNames ...string) {
```

File: service/rocketchat/rocketchat.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	r.chNames = append(r.chNames, chNames...)
+}
+
+// Send takes a message subject and a message body and sends them to all previously set channels.
+// user used for sending the message has to be a member of the channel.
+// https://docs.rocket.chat/api/rest-api/methods/chat/postmessage
+func (r *RocketChat) Send(ctx context.Context, subject, message string) error {
+	fullMessage := subject + "\n" + message // Treating subject as message title
+
+	for _, chName := range r.chNames {
```
Comment (nikoksr) [inline service/rocketchat/rocketchat.go:56]:
```suggestion
	for _, channelName := range r.channelNames {
```

File: service/rocketchat/rocketchat.go (hunk 1 above)
Comment (nikoksr) [reply service/rocketchat/rocketchat.go:15]: Please apply for the remaining use cases.

**Review by nikoksr — CHANGES_REQUESTED** [review]
@kha7iq great job, very much appreciate your contribution. Please just apply my minor style recommendations.

**Comment by kha7iq** [comment]
@nikoksr  I have update the code with style recommendations , do let me know if there is anything else.
And thank you for your work

**Review by nikoksr — APPROVED** [review]
Appreciate your work and kind words :rocket:

**Pull request merged**